torchdynamo/convert_frame.py

```diff
@@ -172,16 +176,35 @@ def has_tensor(obj):
         if has_tensor(value):
             return True
 
-    if config.debug:
-        print(
-            "skipping because no torch.*",
-            frame.f_code.co_name,
-            frame.f_code.co_filename,
-            frame.f_code.co_firstlineno,
-        )
+    log_debug(
+        f"skipping because no torch.* {frame.f_code.co_name} \
+            {frame.f_code.co_filename} {frame.f_code.co_firstlineno}"
+    )
+
     return False
 
 
+def format_exception(exc, frame):
+    msg = os.linesep * 2
```

Comment:
Can we also print the type of exception - SkipFrame vs TorchDynamoInternal etc

Reply:
I believe traceback.format_exc already does this